src/core/audio/AudioSdl.cpp

```diff
@@ -37,6 +38,10 @@
 namespace lmms
 {
 
+constexpr char const* const c_sectionSDL = "audiosdl";
```

Comment:
constexpr const* const ? The level of const here doesn't make sense

Reply:
It does make sense. It's also a nice example to demonstrate why I like to put the `const` to the right of the thing that should be declared `const`. If written like that you can read the line from right to left to find out what it declares: "`c_sectionSDL` is a constant pointer to constant characters. The whole thing is a `constexpr` which means that it can be computed/evaluated at compile time by the compiler."

So I declare that the pointer must not be changed to point to some other object and that the characters the pointer points at also must not be changed.

Reply:
For object declarations, `constexpr` implies `const`, so the second `const` can be omitted. Also, `char const*` can be replaced with `auto`. This results in the equivalent declaration `constexpr auto c_sectionSDL = "audiosdl";`, which I think is a bit clearer.

Reply:
Fixed with commit https://github.com/LMMS/lmms/pull/7421/commits/f3b284b5005b8b7680674b73da3f009db5775713.